### PR TITLE
Added logging for play event

### PR DESCRIPTION
**Pull request by rodrigopavezi**
@donnapep @stulees please review. cheers

### CODE REVIEW

File: test/integration/logging.html [hunk 1]
```diff
@@ -58,8 +60,15 @@
...
 
-    teardown(function() {
-      RiseVision.Common.LoggerUtils.logEvent.restore();
+    setup(function() {
+    });
+
+    suite("play", function() {
+      test("should log the play event", function() {
+        params.event = "play";
+        assert(spy.calledTwice);
```
Comment (donnapep) [inline test/integration/logging.html]: @rodrigopavezi Just wondering why this would be getting called twice?

Comment (rodrigopavezi) [inline test/integration/logging.html]: @donnapep because of the logging of the x-frame-error. cheers

Comment (donnapep) [inline test/integration/logging.html]: @rodrigopavezi Right, makes sense.

**Comment by donnapep** [comment]
👍

**Pull request merged**